jdbc/driver-appengine-standard/src/test/java/com/google/cloud/trace/jdbc/AppEngineTraceServiceTest.java

```diff
@@ -56,7 +56,7 @@ public void exampleSpan() {
 
     InOrder inOrder = inOrder(mockTraceService, mockSpan);
     inOrder.verify(mockTraceService).startSpan("JDBC.Driver#connect");
-    inOrder.verify(mockSpan).setLabel("trace.cloud.google.com/jdbc/url", "jdbc:mockdriver");
+    inOrder.verify(mockSpan).setLabel("g.co/jdbc/url", "jdbc:mockdriver");
```

Comment:
Ha! Excellent point! However it seems that the team has settled on using g.co. ;-)